Cargo.toml

```diff
@@ -25,7 +25,7 @@ runtime = []
 
 [dependencies]
 serde = { version = "1.0.127", features = ["derive"] }
-anyhow = "1.0.42"
+thiserror = "1.0"
```

Comment:
Let's use the patch release here, too. Usually dependabot should take care of the updates.